app/formSchema/uiSchema/pages/submission.ts

```diff
@@ -1,14 +1,19 @@
+import { ReadOnlySubmissionWidget } from '../../../lib/theme/widgets';
+
 const submission = {
   'ui:order': [
     'submissionCompletedFor',
-    'submissionDate',
     'submissionCompletedBy',
     'submissionTitle',
+    'submissionDate',
   ],
   'ui:title': '',
   'ui:field': 'SubmissionField',
+  submissionCompletedFor: {
+    'ui:widget': ReadOnlySubmissionWidget,
+  },
   submissionDate: {
-    'ui:widget': 'DatePickerWidget',
+    'ui:widget': ReadOnlySubmissionWidget,
```

Comment:
```suggestion
    'ui:widget': 'ReadOnlySubmissionWidget',
```

Widgets should be strings and added to the theme, otherwise themes won't work